doc/user-guide/Push-notifications.md

```diff
@@ -158,7 +160,7 @@ The client sends the following stanza to the server:
 </iq>
 ```
 
-The `pubsub.mypubsub.com` will be used as an gateway for all notifications and will pass them through to the APNS and/or FCM. 
+The `pubsub.mypubsub.com` will be used as an gateway for all notifications and will pass them through to the APNS and/or FCM.
```

Comment:
The `pubsub.mypubsub.com` will be used as a gateway for all notifications and will pass them through to the APNS and/or FCM.